targets/f7/api_symbols.csv

```diff
@@ -1695,6 +1696,7 @@ Function,+,furi_timer_start,FuriStatus,"FuriTimer*, uint32_t"
 Function,+,furi_timer_stop,FuriStatus,FuriTimer*
 Function,-,fwrite,size_t,"const void*, size_t, size_t, FILE*"
 Function,-,fwrite_unlocked,size_t,"const void*, size_t, size_t, FILE*"
+Function,+,gallagher_deobfuscate_and_parse_credential,void,"GallagherCredential*, const uint8_t*"
```

Comment:
Everything in API table is linked into firmware image. Such specific piece of code belongs to the app/plugin that makes use of it, so better move it there.

Reply:
I put this function in `lib/nfc/helpers/gallagher_util.c`, because I wanted to make it reusable but still keep it in the content of being used with the NFC app. It was placed in the API symbols table by the build system. Can you suggest where I might move it?

I want to keep it somewhere it can be reused, because I've only provided MIFARE Classic support, but the DESFire cards contain the same type of credential (once unlocked and read).